pkg/ddevapp/ddevapp.go

```diff
@@ -2928,20 +2928,18 @@ func (app *DdevApp) Stop(removeData bool, createSnapshot bool) error {
 			vols = append(vols, app.Name+"-ddev-config")
 		}
 		for _, volName := range vols {
-			err = dockerutil.RemoveVolume(volName)
-			if err != nil {
-				util.Warning("Could not remove volume %s: %v", volName, err)
-			} else {
-				util.Success("Volume %s for project %s was deleted", volName, app.Name)
+			if dockerutil.VolumeExists(volName) {
```

Comment:
It's possible this change could have been done in RemoveVolume() itself, but no need to change. I imagine you considered that.

Reply:
Yes, it can be moved, but let's leave it as it is.